### DISCUSSION TITLE
Smokeping active?

### DISCUSSION THREAD

**Discussion by grguthrie** [oetiker/SmokePing#351] [Q&A] [answered]
I have an odd issue on a reinstall to a new machine, and tried to ask on the mailing list, but it bounced - undeliverable, and looking online it seems like the last postings were in 2018.
Is three any activity or support for this (wonderful!) tool?

**Comment by oetiker** [comment, marked as answer]
yes :) here  ... updating the website is one of the tasks which keeps falling by the wayside :(

**Comment by moetiker** [comment]
Yes, it is still active ... don't worry

**Comment by grguthrie** [comment]
Neat – great – thanks!

I have an error with a transplanted setup, where might I seek some help?

Thanks for a great tool!

Gregory Guthrie

**Reply by Strykar** [reply]
Create a new Issue / Discussion about it.
